### ISSUE TITLE
Implement shared user account

### ISSUE THREAD

**Issue by AlenPelin** [Sitecore/Sitecore-Instance-Manager#27]

**Comment by bchristie** [comment]
Are you planning on storing settings in Environment.CommonAppConfig? Any ideas on this, as I'd like to stand up SIM as a service (rest or wcf) and allow it to be run headless.

**Comment by bchristie** [comment]
And apologies, logged into personal git.

/cc @brad-christie

**Comment by AlenPelin** [comment]
@bchristie this wish was about creating shared Sitecore user account so just after fresh Sitecore installation you can log in as sitecore\brad with password ScRulez! instead of sitecore\admin.

**Comment by Brad-Christie** [comment]
Is this account an amin account?  (I assume so)

Brad Christie
Sitecore
Solutions Architect
55 S. Commercial St., Manchester, NH 03103

**Comment by AlenPelin** [comment]
Yes, it should be convenient to have your own account with your email, password etc. in all instances you install, with your desktop background preferences, icons on desktop etc.

**Comment by Brad-Christie** [comment]
Where is "brad" coming from, AdvancedSetting? Examine logged in user? Also, should desktop background (and other prefs) also be AdvancedSettings?

**Comment by AlenPelin** [comment]
Yes, AdvancedSetting should be good place for that as well as desktop background and other preferences.

**Comment by Brad-Christie** [comment]
Have this done, just haven't checked in yet--presales takes priority. Will check in when I have a moment to catch my breath.
